src/ranged.cpp

```diff
@@ -603,9 +603,12 @@ bool ranged::handle_gun_damage( Character &shooter, item &it )
     }
 
     const auto &curammo_effects = it.ammo_effects();
-    const islot_gun &firing = *it.type->gun;
+    // If using an aux magazine for a gun, make the parent gun accumulate and track fouling instead of the gunmod that's doing the firing.
+    item &parent = it.parent_item() != nullptr &&
+                   it.has_flag( flag_USE_PARENT_GUN ) ? *it.parent_item() : it;
```

Comment:
```suggestion
    item &parent = ( it.parent_item() != nullptr &&
                   it.has_flag( flag_USE_PARENT_GUN ) ) ? *it.parent_item() : it;
```
just makes it a little more explicit that the ternary is for that entire preceding condition.